imixs-workflow-core/src/main/java/org/imixs/workflow/util/XMLParser.java

```diff
@@ -396,6 +378,28 @@ public static ItemCollection parseTag(String xmlContent, String tag) throws Plug
         return result;
     }
 
```

Comment:
Can you please add the following method comment
and remove the out-comment old code parts. 

/** 
 * This helper method parses all child elements and creates / or appends the tag value as a Item value into the result ItemCollection. 
 * The method is called from parseTag.
 **/

Reply:
Thanks for the review, added the mentioned comment and removed commented code from the helper method.

Reply:
Great!